src/cart/crt_internal_types.h

```diff
@@ -150,6 +150,8 @@ struct crt_gdata {
 	unsigned int             cg_provider_is_primary : 1;
 	/** use legacy progress method */
 	bool                     cg_progress_legacy;
+	/** use single thread to access context */
```

Comment:
might want to add new field to be dumped in crt_gdata_dump(void). even though we dump envs automatically, someone might set new field via init_opt, and would be good to see that in logs.